apps/admin-gui/src/app/admin/pages/admin-page/admin-owners/admin-owners.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit } from '@angular/core';
+import { Owner, OwnersManagerService } from '@perun-web-apps/perun/openapi';
+import { TABLE_GROUP_RESOURCES_LIST, TableConfigService } from '@perun-web-apps/config/table-config';
+import { MatDialog } from '@angular/material/dialog';
+import { GuiAuthResolver } from '@perun-web-apps/perun/services';
+import { SelectionModel } from '@angular/cdk/collections';
+import { PageEvent } from '@angular/material/paginator';
+import { getDefaultDialogConfig } from '@perun-web-apps/perun/utils';
+import { DeleteOwnerDialogComponent } from '../../../../shared/components/delete-owner-dialog/delete-owner-dialog.component';
+import { AddOwnerDialogComponent } from '../../../../shared/components/add-owner-dialog/add-owner-dialog.component';
+
+@Component({
+  selector: 'perun-web-apps-admin-owners',
```

Comment:
You also should change the selector prefix to `app` instead of `perun-web-apps`.